### PR TITLE
minor fixes on staking pages

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

Fixes typos and standardizes terminology across staking pages

### CODE REVIEW

File: src/content/staking/withdrawals/index.md [hunk 1]
```diff
@@ -148,12 +148,12 @@ As an alternative to changing the withdrawal address for a particular validator,
 </ExpandableCard>
 
 <ExpandableCard
-title="What if I participate in liquid staking derivatives or pooled staking"
+title="What if I participate in  staking tokens or pooled staking"
```
Comment (minimalsm) [inline src/content/staking/withdrawals/index.md:151]:
```suggestion
title="What if I participate in staking tokens or pooled staking"
```

Comment (minimalsm) [reply src/content/staking/withdrawals/index.md:151]: Note: maybe should add a check for double-spacing in the markdown checker?

File: src/content/staking/solo/index.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ It also requires very basic hardware setup, and some understanding of minimum re
 <ExpandableCard title="Secure key management" eventCategory="SoloStaking" eventName="clicked secure key management">
 Just like how private keys secure your Ethereum address, you will need to generate keys specifically for your validator. You must understand how to keep any seed phrases or private keys safe and secure.
 
-<a href="/security/">Ethereum security and scam prevention</a>
+<a href="/security/"> Ethereum security and scam prevention</a>
```
Comment (minimalsm) [inline src/content/staking/solo/index.md:58]: Why did we add a space here?

```suggestion
<a href="/security/">Ethereum security and scam prevention</a>
```

**Review by minimalsm — APPROVED** [review]
LGTM, other than the two spacing changes I highlighted here.

Should probably let @wackerow once over this as the original author, but all looks good on my end.

File: src/content/staking/solo/index.md (hunk 2 above)
Comment (jmcook1186) [reply src/content/staking/solo/index.md:58]: because on the rendered page the link buts up against the preceding text

**Review by corwintines — APPROVED** [review]

File: src/content/staking/solo/index.md (hunk 2 above)
Comment (wackerow) [reply src/content/staking/solo/index.md:58]: We should add `{' '}` before instead

Comment (wackerow) [reply src/content/staking/solo/index.md:58]: Or a `<br/>` or two to bring it down to a new line

Comment (corwintines) [reply src/content/staking/solo/index.md]: Pushed up a change to add {' '}

This addresses the issues above.

**Review by corwintines — APPROVED** [review]

**Pull request merged**